src/actions/guides/getting-started/installing.cr

```diff
@@ -77,7 +77,7 @@ class Guides::GettingStarted::Installing < GuideAction
     * cd into the newly cloned directory
     * Check out latest [released version](https://github.com/luckyframework/lucky_cli/releases) `git checkout #{LuckyCliVersion.current_tag}`
     * Run `shards install`
-    * Run `crystal build src/lucky.cr`
+    * Run `crystal build --release src/lucky.cr`
```

Comment:
```suggestion
    * Run `crystal build src/lucky.cr`
```

Reply:
OMG. github... Apparently I can't commit that suggestion, and it keeps throwing an error when I try to unapprove 😝